StRoot/StSpinPool/StFcsTriggerSimMaker/StFcsTriggerSimMaker.h

```diff
@@ -42,13 +46,19 @@ class StFcsTriggerSimMaker : public StMaker{
     void setEtGain(float v) {mEtFactor=v;} 
 
     fcs_trg_base* getTriggerEmu() {return mTrgSim;}
+    
+    void feedADC(StFcsHit* hit, int ns, int ehp, uint16_t data_array[]);
+    void feedADC(StMuFcsHit* hit, int ns, int ehp, uint16_t data_array[]);
```

Comment:
```suggestion
    template<typename T>
    void feedADC(T* hit, int ns, int ehp, uint16_t data_array[]);
```

Reply:
Great suggestion. Just made the change let me know if that checkout